backend/migration_service.py

```
@@ -1,950 +1,266 @@
+                ssh_port=request.ssh_port,
+                source_host=request.source_host,
+                source_ssh_user=request.source_ssh_user or "root",
+                source_ssh_port=request.source_ssh_port or 22,
+                force_rsync=request.force_rsync or False
+            )
+            
+            return await self.start_container_migration(container_request)
+            
+        except Exception as e:
+            logger.error(f"Failed to convert legacy migration to container migration: {e}")
+            raise ValueError(f"Legacy migration failed: {e}. Please use container migration instead.")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)